File: KantechGatewayApp/Jobs/DeactivateProcessor.vb
Namespace KantechGatewayApp.Jobs
    Public Class DeactivateProcessor
        Inherits BaseProcessor
        Protected Overrides Sub HandleRow(row As Dictionary(Of String, String))
            Dim cardId = row.GetValueOrDefault("Card ID")
            Dim empId = row.GetValueOrDefault("employee ID")
            Infrastructure.Logger.Info($"[Deactivate] card={cardId} emp={empId} -> deactivated", _logSub)
        End Sub
    End Class
End Namespace

File: KantechGatewayApp/Jobs/DeactivateProcessor.cs
using System.Collections.Generic;

namespace KantechGatewayApp.KantechGatewayApp.Jobs
{
    public class DeactivateProcessor : BaseProcessor
    {
        protected override void HandleRow(Dictionary<string, string> row)
        {
            string cardId = row.GetValueOrDefault("Card ID");
            string empId = row.GetValueOrDefault("employee ID");

            Infrastructure.KantechJobApi.TriggerJobApi(_jobKey, row);
            Infrastructure.Logger.Info($"[Deactivate] card={cardId} emp={empId} -> deactivated", _logSub);
        }
    }
}